roles/kubernetes/control-plane/tasks/kubeadm-upgrade.yml

```diff
@@ -27,8 +27,7 @@
   environment:
     PATH: "{{ bin_dir }}:{{ ansible_env.PATH }}"
 
-  # TODO: Remove --skip-phases from command when v1beta4 UpgradeConfiguration supports skipPhases
```

Comment:
Hum, why remove that todo though ? Is there a reason to not switch to the config file once it works ?

Reply:
I removed the comment from the kubeadm upgrade node task because it's inaccurate. skipPhases has always worked for upgrade node.

Reply:
@VannTen Added a TODO comment with a reminder to remove effectively all args except for `--config` from `kubeadm upgrade node` when we drop support for v1beta3. Please take a look. Thanks!